Dockerfile

```diff
@@ -126,11 +126,16 @@ RUN cd /tmp && \
 
 
 # Final stage
-FROM ubuntu:22.04
+FROM ubuntu:24.04
 LABEL maintainer="sysadmins@cs50.harvard.edu"
 ARG DEBIAN_FRONTEND=noninteractive
 
 
+# Remove exisiting ubuntu user (if any) and home directory
+RUN userdel --force --remove ubuntu && \
+    rm --force --recursive /home/ubuntu
+
```

Comment:
@dmalan  Fyi, Ubuntu 24.04 will have a default `ubuntu` user created with some `.bashrc` files located in `/home/ubuntu`. It is cleanest to just delete this default ubuntu user and remove any files under `home` before doing anything.